client/src/components/Homepage/tickets-home-my.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment } from 'react';
+import Navbar from '../navbar/Navbar';
+import './style.css';
+import MainSidebar from '../MainSidebar';
+
+export default ({ status }) => (
+  <Fragment>
+    <MainSidebar selected="tickets" />
+    <Navbar logoutClass="home-logout" className="homeNav">
```

Comment:
why are we giving different classes to the logout component? Is it going to change from a view to another? if not, then it's not necessary.